### PR TITLE
feat: add registry-instance to generate script

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: scripts/generate.sh [hunk 1]
```diff
@@ -89,3 +89,22 @@ npx @openapitools/openapi-generator-cli generate -g java \
     --package-name="${PACKAGE_NAME}" \
     --additional-properties="apiTests=false,modelTests=false,hideGenerationTimestamp=true,groupId=${GROUP_ID},artifactId=${ARTIFACT_ID},modelPackage=${PACKAGE_NAME}.models,invokerPackage=${PACKAGE_NAME}.invoker,apiPackage=${PACKAGE_NAME},dateLibrary=java8,licenseName=Apache-2.0,licenseUrl=https://www.apache.org/licenses/LICENSE-2.0.txt" \
     --ignore-file-override=.openapi-generator-ignore
+
+GROUP_ID="com.redhat.cloud"
+OPENAPI_FILENAME=".openapi/registry-instance.json"
+ARTIFACT_ID="registry-instance-sdk"
+OPENAPI_FILENAME=".openapi/registry-instance.json"
+PACKAGE_NAME="com.openshift.cloud.api.registry.instance"
```
Comment (dimakis) [inline scripts/generate.sh:97]: is this the correct naming structure? @wtrocki

Comment (wtrocki) [reply scripts/generate.sh:113]: Looks good

**Comment by wtrocki** [comment]
We are missing code changes from generator. Otherwise LGTM

**Comment by dimakis** [comment]
> We are missing code changes from generator. Otherwise LGTM

sorry, I don't get what you mean here. code is generated by GH actions?

**Pull request merged**